src/components/form/show.vue

```diff
@@ -99,8 +99,8 @@ const stopAppUsersEffect = watchEffect(() => {
   nextTick(() => stopAppUsersEffect());
 });
 const stopDatasetsEffect = watchEffect(() => {
-  if (!(project.dataExists && form.dataExists)) return;
-  if (project.permits(['form.update', 'dataset.list']) && form.publishedAt != null)
+  if (!(form.dataExists)) return;
```

Comment:
Might be able to get rid of one set of parens?

```suggestion
  if (!form.dataExists) return;
```